roles/runtime/tasks/main.yaml

```diff
@@ -48,18 +48,18 @@
         mode: '0644'
     - name: Install crictl - {{ critools_version }}
       unarchive:
-        src: "https://github.com/kubernetes-sigs/cri-tools/releases/download/{{ critools_version }}/crictl-{{ critools_version }}-linux-ppc64le.tar.gz"
+        src: "https://github.com/kubernetes-sigs/cri-tools/releases/download/v{{ critools_version }}/crictl-v{{ critools_version }}-linux-ppc64le.tar.gz"
         dest: "/usr/local/bin/"
         remote_src: yes
 
     - name: Install iptables
       yum:
         name: iptables
 
-    - name: Install container runtime - runc
+    - name: Install runc - {{ runc_version }}
       get_url:
         url: "https://github.com/opencontainers/runc/releases/download/v{{ runc_version }}/runc.ppc64le"
-        dest: /usr/bin/runc
+        dest: /usr/local/bin/runc
```

Comment:
Clubbing the binary to the same path shared by others that are installed during the installation of k8s.

Reply:
hope we don't need to update this path anywhere else for the runs?!

Reply:
I feel this is already accounted as default $PATH is /usr/local/sbin:/usr/local/bin:/usr/sbin:/usr/bin:/root/bin and runc should be globally available. The change didn't impact the functioning of cluster when tested..

I did check if there are tests that may refer to the path changed, but looks like there are none.